versioning/src/main/scala/org/typelevel/sbt/TypelevelVersioningPlugin.scala

```diff
@@ -82,14 +82,18 @@ object TypelevelVersioningPlugin extends AutoPlugin {
               None
           }
 
+        // version here is the prefix used further to build a final version number
         var version = latestInSeries.fold(tlBaseVersion.value)(_.toString)
 
-        // Looks for the distance to latest release in this series
-        latestInSeries.foreach { latestInSeries =>
-          Try(s"git describe --tags --match v$latestInSeries".!!.trim)
-            .collect { case Description(distance) => distance }
-            .foreach { distance => version += s"-$distance" }
-        }
+        if (version < baseV.toString)
```

Comment:
Comparing versions by comparing `String` makes me nervous 😅 does this do what we need? Esp. since `version` is in the `x.y.z` form and `baseV` is in the `x.y` form.

Reply:
Btw, the `V` model has some comparison capability.
https://github.com/typelevel/sbt-typelevel/blob/5fff779cd21c5c2e81eb667d9bfb9b0f42b97cc6/kernel/src/main/scala/org/typelevel/sbt/kernel/V.scala#L21-L26

Reply:
I totally forgot about that till I check the values in repl

two options on how I see it working 

- version.take(3) < baseV.toString that would give us the version in `x.y` form or
- we call apply method of V with the string and then pattern match on the value we get back.

WDYT ?

Reply:
`version.take(3)`: what if `version` is `1.23`?

I like option 2 :) `V` is supposed to help working with versions strings.

One thing to be careful of: `V("x.y") > V("x.y.z")` for all `z`. This may not have been a good idea, but I originally designed it this way so the `baseVersion` is always in the "future". Actually I'm not sure if this is used anywhere anymore.